src/reducers/files-and-folders-metadata/files-and-folders-metadata-reducer.test.ts

```diff
@@ -0,0 +1,31 @@
+import { initFilesAndFoldersMetatada } from "./files-and-folders-metadata-actions";
+import reducer from "./files-and-folders-metadata-reducer";
+import { FilesAndFoldersMetadataState } from "./files-and-folders-metadata-types";
+
+const initialState: FilesAndFoldersMetadataState = {
+  filesAndFoldersMetadata: {}
+};
+
+describe("files-and-folders-metadata-reducer", () => {
+  describe("INIT_FILES_AND_FOLDERS_METADATA", () => {
+    it("should set the metadata", () => {
```

Comment:
`it("should set the metadata on filesAndFolders")` pour être consistant avec les autres tests